spec/controllers/application_controller_spec.rb

```diff
@@ -58,9 +58,14 @@ def not_found
   end
 
   shared_examples_for "respects Authorization header" do
+    let(:key)   { Keycard::DigestKey.new }
+    let!(:user) { Fabricate(:user, admin: true, api_key_digest: key.digest ) }
+
     context "with valid Authorization header" do
-      let(:user) { Fabricate(:user, admin: true) }
-      let(:auth_header) { { "Authorization" => "Token token=#{user.api_key}" } }
+      # TODO: This all goes away once we take the authenticate_with_http_token
+      # built-in out. We have to explicitly access the user, even when defined with let!,
+      # because it is not saved before the controller method executes otherwise... shrug
+      let(:auth_header) { user; { "Authorization" => "Token token=#{key.to_s}" } }
```

Comment:
Whoops. I left my rough comment in here. It's not really that it "all" goes away since we are planning to use the Authorization header.. What _might_ go away is the immediate access of the user to coerce the timing vs. controller action. I could not figure out why it was necessary here but not in other spots.